AppCenterAnalytics/AppCenterAnalyticsTests/MSACSessionTrackerTests.m

```diff
@@ -142,6 +142,82 @@ - (void)testLongBackgroundSession {
   XCTAssertNotEqual(expectedSid, sid);
 }
 
+- (void)testStartSessionWhenManualSessionTrackerEnabled {
+
+  // Stop session.
+  [self.sut stop];
+
+  // When.
+  [self.sut enableManualSessionTracker];
+
+  // Start new manual session.
+  [self.sut startSession];
+  MSACLogWithProperties *firstManualLog = [MSACLogWithProperties new];
+  [self.sut channel:nil prepareLog:firstManualLog];
+  NSString *previousSid = firstManualLog.sid;
```

Comment:
you don't need to store it in a variable, you can always use `firstManualLog.sid` in your checks.